### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by katosdev** [sysadminsmedia/homebox#96] [Announcements]
## Overview
This release introduces numerous bug fixes and UI improvements, but also introduces new functionality such as total pricing on locations, introduction of more currencies, Docker healthcheck and more.

## What's Changed
* Handle redirect to correct page after login by @harrisony in https://github.com/sysadminsmedia/homebox/pull/76
* feat(docs): i18n support added to docs by @tankerkiller125 in https://github.com/sysadminsmedia/homebox/pull/69
* Add total pricing to Homebox locations by @katosdev in https://github.com/sysadminsmedia/homebox/pull/42
* Move the link tooltip to the top to prevent overflows by @zebrapurring in https://github.com/sysadminsmedia/homebox/pull/77
* Implement filter option for negated labels by @zebrapurring in https://github.com/sysadminsmedia/homebox/pull/70
* Update currencies to include more currencies by @katosdev in https://github.com/sysadminsmedia/homebox/pull/81
* Update README.md by @tankerkiller125 in https://github.com/sysadminsmedia/homebox/pull/83
* chore(deps): bump github.com/gorilla/schema from 1.2.1 to 1.4.1 in /backend in the go_modules group across 1 directory by @dependabot in https://github.com/sysadminsmedia/homebox/pull/87
* chore: General cleanup tasks by @tankerkiller125 in https://github.com/sysadminsmedia/homebox/pull/88
* Add Dockerfile healthcheck by @katosdev in https://github.com/sysadminsmedia/homebox/pull/78
* Rectify Docker build errors due to broken imports by @katosdev in https://github.com/sysadminsmedia/homebox/pull/93
* fix(ci): remove linting temp, and always build on docker changes by @tankerkiller125 in https://github.com/sysadminsmedia/homebox/pull/94
* Update WGET from Busybox to APK by @katosdev in https://github.com/sysadminsmedia/homebox/pull/95

## New Contributors
* @harrisony made their first contribution in https://github.com/sysadminsmedia/homebox/pull/76

**Full Changelog**: https://github.com/sysadminsmedia/homebox/compare/v0.11.1...v0.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/sysadminsmedia/homebox/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by IwishIcanFLighT** [comment]
Thanks for the great work and this release. Looking forward to the next! 👍 

I just updated my server to test the location pricing feature. It seems like it doesn't consider the Quantity in the calculation. Here's an example:

![image](https://github.com/sysadminsmedia/homebox/assets/4090502/15a174b0-485f-4e8b-ad90-539e1f5f3ae9)

- Item A : 2 Qt * 1€ = 2€
- Item B : 5 Qt * 5€ = 25€
- Total : 2 + 25 = 27€ instead of 6€

Am I missing something, or is it actually the way it's intended? Because I believe the Total Value in the Homepage dashboard considers the quantity.

**Reply by tankerkiller125** [reply]
@katosdev wrote this feature, I think you are correct that it's not taking quantity into account.

**Reply by IwishIcanFLighT** [reply]
Thanks for your reply! I think it should be taking quantity into account imho. Otherwise, the total value is inaccurate for locations containing packs/groups of items. Besides, it doesn't affect single items.

**Reply by katosdev** [reply]
How strange! My apologies for this, can you open an issue please and I’ll have a look at that!

**Reply by katosdev** [reply]
@IwishIcanFLighT 

I noticed an issue was not opened. I will get one opened and investigate this.

**Reply by katosdev** [reply]
Fixed in #123

**Reply by IwishIcanFLighT** [reply]
Awesome, thank you very much! And sorry I couldn't submit the issue earlier. 😅
